ngx-tools/jwt/testing/src/retry-with-escalation.mock.ts

```diff
@@ -0,0 +1,71 @@
+// tslint:disable: no-any
```

Comment:
NOTE: These JWT mocks look like brand new code, but they are simply copied from the `testing` dir (they will be removed from the `testing` endpoint at a later date)